Added Default favicon

ddomdel opened the pull request:
I added a default favicon in the static folder. Let me know if this is the right one or if I added it correctly. Thanks!

This is related to this issue: https://github.com/CTFd/core-beta/issues/8

ColdHeat commented on templates/base.html:
```diff
@@ -4,6 +4,7 @@
   <title>{{ Configs.ctf_name }}</title>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="shortcut icon" href="{{ url_for('views.themes', theme='core', path='img/favicon.ico') }}" type="image/x-icon">
```
i think the endpoint should be `views.themes_beta` for the core-beta theme for now. Also we should probably be just using the core-beta assets so no need to specify the theme. If you need to you can copy over the favicon file into the assets folder and then modify vite.config.js to copy that into the static folder.

ddomdel replied:
Alright. I modified it now. 

I have a question. I am running this theme inside a local docker container and when you open the site in the browser and check where it got the favicon.ico, it says that it got it from core-beta/static/img folder
![image](https://user-images.githubusercontent.com/78736225/180988677-f32b9584-bb86-4c7c-9a2d-85c131ebde3e.png)
But, when I check the files in VSCode it doesn't have the img folder inside the static folder.
![image](https://user-images.githubusercontent.com/78736225/180988922-9d4718f6-a51d-4e04-8f3d-5099f6389d17.png)

ddomdel commented:
- Added `img `folder to `assets` folder. 
- Configured Vite to copy the contents of `img` folder to `static/assets` folder.
- Modified `<Link/>` href to look into `/static/assets` folder.
![assets-location](https://user-images.githubusercontent.com/78736225/181423120-cb2d87c6-0a87-476e-921d-7b298b271511.png)
Verified if it is really looking into the intended folder to look into, which should be `/themes/core-beta/static/assets/favicon.ico`, by inserting a JS script in `base.html` and letting flask-jinja render it
![js-test-asset-location](https://user-images.githubusercontent.com/78736225/181423423-f148a0bb-61d9-44c6-ba4d-ec463268c0e6.png)
![assets-location1](https://user-images.githubusercontent.com/78736225/181423465-543958b3-88a6-4ea5-89e5-81667cc17063.png)

ColdHeat commented:
The static files I don't think should have been deleted from here. Can you put them back?

ddomdel commented:
Yeah I noticed that too. But I think it happens when vite recreates those static files, it creates a new hash on its name. So git thinks that it has been deleted I think. Not really sure, but I'll restore it instead.

ColdHeat commented on vite.config.js:
```diff
@@ -32,6 +32,10 @@ export default defineConfig({
             },
+            {
+               src: "./assets/img/**",
+               dest: "static/assets",
```
This should be `static/img`

ColdHeat commented:
This isn't right because you can't override it from the Admin Panel. 

See https://github.com/CTFd/CTFd/blob/master/CTFd/themes/core/templates/base.html#L7.
